ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
```

Comment:
The pull request title and description appear to be empty. Could you please provide a meaningful title and description to better understand the purpose of these changes?